index.js

```diff
@@ -1,16 +1,19 @@
-'use strict';
-const {promisify} = require('util');
-const path = require('path');
-const childProcess = require('child_process');
-const fs = require('fs');
-const isWsl = require('is-wsl');
-const isDocker = require('is-docker');
-
-const pAccess = promisify(fs.access);
-const pReadFile = promisify(fs.readFile);
+import path from 'path';
+import childProcess from 'child_process';
+import {promises as fs} from 'fs';
+import {fileURLToPath} from 'url';
+import isWsl from 'is-wsl';
+import isDocker from 'is-docker';
+import defineLazyProperty from 'define-lazy-prop';
+import AggregateError from 'aggregate-error';
+
+// Node.js ESM doesn't expose __dirname (https://stackoverflow.com/a/50052194/8384910)
```

Comment:
Can't tell whether this will affect

https://github.com/sindresorhus/open/blob/02bff0124bed4a8841ce8ea92bd44ddd34cd9e2a/index.js#L135-L136